README.md

```diff
@@ -42,10 +42,10 @@ This section is **in progress** here: [New contributor Docs - Contributing](http
 
 ### Useful Links
 
-* CoreFX source index: https://source.dot.net
-* API Reference docs: https://docs.microsoft.com/en-us/dotnet/core/api
-* .NET API Catalog: http://apisof.net (incl. APIs from daily builds and API usage info)
-* "Full" .NET Framework source index: https://referencesource.microsoft.com
+* [CoreFX source index](https://source.dot.net)
+* [API Reference docs](https://docs.microsoft.com/en-us/dotnet/core/api)
+* [.NET API Catalog (incl. APIs from daily builds and API usage info)](http://apisof.net) 
+* ["Full" .NET Framework source index](https://referencesource.microsoft.com)
```

Comment:
Unrelated to this PR: does it make sense to call .Net Framework "Full"? Especially considering that .Net Standard 2.0 brought a lot of APIs back and that .Net Framework in now lagging behind .Net Core when it comes to new APIs.

Reply:
We usually call it just ".NET Framework" (vs ".NET Core") as those are the correct names although possibly this is confusing to come people.